network_wrangler/params.py

```diff
@@ -165,11 +165,17 @@ def unique_ids(self) -> list[str]:
     "applied_projects",
     "conflicts",
 ]
-TRANSIT_CARD_TYPES = ["transit_property_change"]
+TRANSIT_CARD_TYPES = [
+    "transit_property_change",
+    "transit_routing_change",
+    "transit_route_addition",
+    "transit_service_deletion"
+]
 ROADWAY_CARD_TYPES = [
     "roadway_deletion",
     "roadway_addition",
     "roadway_property_change",
+    "pycode"
```

Comment:
add `pycode` to roadway card types, but how to deal with `pycode` in transit card (we don;t have transit wrangler card from met council though)?